tests/models/test_DurableOrchestrationContext.py

```diff
@@ -100,3 +100,19 @@ def test_get_input_json_str():
     result = context.get_input()
 
     assert 'Seattle' == result['city']
+
+def test_version_equals_version_from_first_execution_started_event():
+    builder = ContextBuilder('test_function_context')
+    builder.history_events = []
+    builder.add_orchestrator_started_event()
+    builder.add_execution_started_event(name="TestOrchestrator", version="1.0")
+    builder.add_execution_started_event(name="TestOrchestrator", version="2.0")
+    context = DurableOrchestrationContext.from_json(builder.to_json_string())
+    assert context.version == "1.0"
+
+def test_version_is_none_if_no_execution_started_event():
```

Comment:
Remove this test - it's not valid. There will always be an execution_started event in the history when the `context.version` API is called.